roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -101,13 +101,13 @@ class RooDataHist : public RooAbsData, public RooDirItem {
 
   void weights(double* output, RooSpan<double const> xVals, int intOrder, bool correctForBinSize, bool cdfBoundaries);
   /// Return weight of i-th bin. \see getIndex()
-  double weight(std::size_t i) const { return _wgt[i]; }
+  double weight(std::size_t i) const { return get_wgt(i); }
```

Comment:
```suggestion
  double weight(std::size_t i) const { return _wgt[i]; }
```
Why does this need to be re-implemented in terms of `get_wgt`? It would only make sense if `get_wgt` is virtual I guess, but it was not virtual before and it should also not be virtual now if you add it back.